### PR TITLE
connectors: extended Iceberg connector doc

### PR DESCRIPTION
**Pull request by bochenekmartin**

### CODE REVIEW

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/create-iceberg-sink-connector.adoc [hunk 1]
```diff
@@ -10,12 +10,12 @@ You can use the Iceberg Sink connector to write data into Iceberg tables.
...
 must:
 
-. https://iceberg.apache.org/concepts/catalog/[Set up an Iceberg catalog^]
+. https://iceberg.apache.org/concepts/catalog/[Set up an Iceberg catalog^].
 . Create the Iceberg connector control topic, which cannot be used by other connectors. For details, see xref:cloud:create-topic.adoc[Create a Topic].
 
 == Limitations
 
-* You can only use the control topic for one Iceberg sink connector at a time.
+* Each Iceberg sink connector has to have its own control topic. The control topic should be created before creating the connector.
```
Comment (Feediver1) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/create-iceberg-sink-connector.adoc]:
```suggestion
* Each Iceberg sink connector must have its own control topic, which you should create before creating the connector.
```

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/create-iceberg-sink-connector.adoc [hunk 2]
```diff
@@ -92,6 +103,69 @@ To enable Apache Kafka to retain the Debezium change event messages in their ori
 
 See also: https://debezium.io/documentation/reference/stable/transformations/event-flattening.html[Debezium New Record State Extraction SMT^]
 
+== Using Apache Spark with Iceberg
+
+Data stored in Iceberg format can be read by different analytical systems like AWS Athena, Snowflake or Apache Spark.
```
Comment (Feediver1) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/create-iceberg-sink-connector.adoc]:
```suggestion
Data stored in Iceberg format can be read by different analytical systems like AWS Athena, Snowflake, or Apache Spark.
```

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/create-iceberg-sink-connector.adoc [hunk 3]
```diff
@@ -92,6 +103,69 @@ To enable Apache Kafka to retain the Debezium change event messages in their ori
...
+
+The following example uses:
+
+* Iceberg REST catalog
+* AWS S3 bucket as a storage for Iceberg files
```
Comment (Feediver1) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/create-iceberg-sink-connector.adoc]:
```suggestion
* AWS S3 bucket as the storage for Iceberg files
```

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/create-iceberg-sink-connector.adoc [hunk 4]
```diff
@@ -92,6 +103,69 @@ To enable Apache Kafka to retain the Debezium change event messages in their ori
...
+* Apache Spark reading the Iceberg data from S3 bucket
```
Comment (Feediver1) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/create-iceberg-sink-connector.adoc]:
```suggestion
* Apache Spark, which reads the Iceberg data from an S3 bucket
```

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/create-iceberg-sink-connector.adoc [hunk 5]
```diff
@@ -27,18 +27,26 @@ To create the Iceberg Sink connector:
...
-| Property | Description
+| Property name | Property key | Description
 
 | `Topics to export`
+| `topics`
 | Comma-separated list of the cluster topics you want to replicate.
 
+| `Topics regex`
+| `topics.regex`
+| Java regular expression of topics to replicate. For example: specify `.\*` to replicate all available topics in the cluster. Applicable only when *Use regular expressions* is selected.
```
Comment (Feediver1) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/create-iceberg-sink-connector.adoc]:
```suggestion
| Java regular expression of topics to replicate. For example: specify `.*` to replicate all available topics in the cluster. Applicable only when **Use regular expressions** is selected.
```

Comment (Feediver1) [reply modules/deploy/pages/deployment-option/cloud/managed-connectors/create-iceberg-sink-connector.adoc]: Formatting issue with asciidoc here. To see how we handled this in another doc:
https://github.com/redpanda-data/docs/commit/32d077a3e1fd080c67c5cb2076bfc1cb456d1116

**Review by Feediver1 — APPROVED** [review]

**Comment by Feediver1** [comment]
Didn't merge in case @tomasz-sadura has feedback. If there is no further feedback, happy to merge. 

Also, just an FYI @bochenekmartin 👍 
- Please create pull requests from branches in the repository, instead of submitting from your fork.
- Netlify doesn't like when we submit PRs from forked repositories. It identifies them as coming from an unrecognized author.

**Review by tomasz-sadura — APPROVED** [review]
LGTM, thanks

**Pull request merged**